pkg/syncer/volume_health.go

```diff
@@ -80,6 +80,22 @@ func csiGetVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 		}
 	}
 
+	if !isVolumeHealth {
```

Comment:
VolumeHealth and pv-to-vvol mapping sounds like two different features. It is not a good idea to piggyback the logic of filling the mapping in the func of `csiGetVolumeHealthStatus`. Please think about it and decouple the implementation of these two features accordingly.

Reply:
Split the pv to vvol implementation into a separate file.